### ISSUE TITLE
Pinata - create folder instead of uploading to the root

### ISSUE THREAD

**Issue by dokluch** [FTM-Labs/Aptos-NFT-Mint#30]
Feature request - it would be beneficial to first create a folder and then upload to it on IPFS, or point config directly to the already created folders

**Comment by bilaljawed** [comment]
> Feature request - it would be beneficial to first create a folder and then upload to it on IPFS, or point config directly to the already created folders

Yeah this would be lovely! need an option to directly point config to the ipfs folder
